components/css/index.css

```diff
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?Montserrat:wght@600&family=Lato:wght@300;400;700&display=swap");
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url("../../docs/css/reset.css");
+@import url("../docs/css/base");
+@import url("../../docs/css/abstracts/colors.css");
+@import url("../../docs/css/abstracts/variables.css");
+@import url("../../docs/css/utilities.css");
+@import url("./typography.css");
+@import url("../../docs/css/navbar.css");
+@import url("../../docs/css/sidebar.css");
+@import url("../../docs/css/doc.css");
+@import url("../../docs/css/footer.css");
+@import url("./avatar.css");
+@import url("./alert.css");
+@import url("./badge.css");
+@import url("./button.css");
+@import url("./card.css");
+@import url("./responsive-img.css");
+@import url("./form.css");
+@import url("./snackbar.css");
+@import url("./navigation.css");
+
+html {
+  font-size: 100%;
+  scroll-behavior: smooth;
+}
+
+body {
+  letter-spacing: 0.025em;
+}
+
+.main_top,
+.main_bottom {
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 2rem 2rem 4rem;
+  padding: 1rem;
+}
+
+.main_bottom__down .code_snippet {
+  width: 60%;
+}
+.main_top h1,
+.main_bottom h1 {
+  font-family: "Montserrat", sans-serif;
+  color: var(--heading);
+  font-weight: 600;
+}
+
+.main_top h1 {
+  font-size: calc(1rem + 3.3vw);
+}
+
+.main_bottom h1 {
+  font-size: calc(1rem + 2vw);
+}
+
+.main_top p,
+.main_bottom p {
+  margin-top: 8px;
+  padding: 8px;
+}
+
+.main_top p {
+  font-size: calc(1.275rem + 0.3vw);
+}
+
+.main_bottom p {
+  font-size: 1.25rem;
+}
+
+.main_top__left {
+  flex: 1;
+  min-width: 300px;
```

Comment:
px to rem conversion

Reply:
yes, but since the fix was going on increasing, I decided to do that in another fix.